File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/receiverQueryApi/model/AudioAccessibilityPref.java
package org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model;

public class AudioAccessibilityPref {
    public VideoDescriptionService videoDescriptionService;
    public AudioEIService    audioEIService;
    public class VideoDescriptionService{
        public boolean enabled;
        public String language;
    }
    public class AudioEIService{
        public boolean enabled;
        public String language;
    }
}

File: app/src/main/java/org/ngbp/jsonrpc4jtestharness/rpc/receiverQueryApi/model/AudioAccessibilityPref.kt
package org.ngbp.jsonrpc4jtestharness.rpc.receiverQueryApi.model

import org.ngbp.jsonrpc4jtestharness.rpc.RpcResponse

data class AudioAccessibilityPref(
        var videoDescriptionService: VideoDescriptionService? = null,
        var audioEIService: AudioEIService? = null
) : RpcResponse()

data class VideoDescriptionService(
        var enabled: Boolean = false,
        var language: String? = null
)

data class AudioEIService(
        var enabled: Boolean = false,
        var language: String? = null
)
